src/containers/Tenant/Diagnostics/Describe/Describe.tsx

```diff
@@ -16,35 +19,60 @@ const b = cn('kv-describe');
 const expandMap = new Map();
 
 interface IDescribeProps {
+    pathType: EPathType;
     tenant: string;
+    schemaNestedChildrenPaths?: string[];
 }
 
-const Describe = ({tenant}: IDescribeProps) => {
+const Describe = ({pathType, tenant, schemaNestedChildrenPaths}: IDescribeProps) => {
     const dispatch = useDispatch();
 
+    const {currentDescribe, error, loading, wasLoaded} = useSelector(
+        (state: any) => state.describe,
+    );
+
     const {
-        currentDescribe: data,
-        error,
-        loading,
-        wasLoaded,
-    } = useSelector((state: any) => state.describe);
-    const {autorefresh, currentSchemaPath} = useSelector((state: any) => state.schema);
-
-    const fetchData = () => {
+        autorefresh,
+        currentSchemaPath,
+        loading: schemaLoading,
+        wasLoaded: schemaWasLoaded,
+    } = useSelector((state: any) => state.schema);
+
+    const hasNested = checkIfPathHasNestedChildren(pathType);
+
+    const fetchData = (resetLoadingState: boolean) => {
         const path = currentSchemaPath || tenant;
-        dispatch(getDescribe({path}));
-    };
 
-    useAutofetcher(fetchData, [], autorefresh);
+        if (hasNested) {
+            // Prevent fetch when shema is not loaded
+            // as in this case component may lack correct children data that is loaded in shema
+            if (!schemaWasLoaded) {
+                return;
+            }
+
+            if (schemaNestedChildrenPaths && schemaNestedChildrenPaths.length > 0) {
+                const paths = [path, ...schemaNestedChildrenPaths];
+                dispatch(
+                    getDescribe({path: paths}, {resetLoadingState, currentDescribePath: path}),
+                );
+            } else {
+                dispatch(getDescribe({path}, {resetLoadingState, currentDescribePath: path}));
+            }
+        } else {
+            dispatch(getDescribe({path}, {resetLoadingState, currentDescribePath: path}));
+        }
+    };
 
-    useEffect(() => {
-        fetchData();
-    }, [currentSchemaPath]);
+    useAutofetcher(
+        (isBackground) => fetchData(!isBackground),
+        [tenant, currentSchemaPath, schemaWasLoaded, schemaNestedChildrenPaths],
+        autorefresh,
```

Comment:
do you have eslint enabled in vscode?
it should warn you about breaking the rules of hooks, which this construction violates. Please follow them

currently linter will warn you about omitting `fetchData` as a dependency, but adding it into the list would make no sense as it is not memoized
but why do you use arrow here and not just `useAutofetcher(fetchData, ...`? This way linter would be more helpful

Reply:
I haven't. Thanks for noticing it, now I set up this extension as I should before.